### PR TITLE
X axis label appears cut off #1475

### PR DESCRIPTION
**Pull request by finnfiddle**
not very elegant but should do the job
- [ ] **Update release notes if necessary**

### CODE REVIEW

File: client/src/components/charts/LineChart.jsx [hunk 1]
```diff
@@ -370,6 +367,10 @@ export default class LineChart extends Component {
                     stroke={'#1b1a1e'}
                     tickTextFill={'#1b1a1e'}
                     numTicks={xAxisTicks}
+                    tickLabelProps={val => ({
+                      transform: `rotate(45, ${xScale(val)}, 18)`,
```
Comment (gabemart) [inline client/src/components/charts/LineChart.jsx]: This is causing a new overlap for date types

image

File: client/src/constants/chart.js [hunk 2]
```diff
@@ -13,4 +13,5 @@ export const connectionStyle = {
 
 export default {
   labelFont,
+  MAX_FONT_SIZE: 12,
```
Comment (gabemart) [inline client/src/constants/chart.js]: On my machine, `MAX_FONT_SIZE` is always `undefined` when imported into the charts

File: client/src/components/charts/LineChart.jsx (hunk 1 above)
Comment (finnfiddle) [reply client/src/components/charts/LineChart.jsx]: try now. this is why I made the original change
screen shot 2018-06-19 at 2 17 20 pm

Comment (gabemart) [reply client/src/components/charts/LineChart.jsx]: For line charts with tick labels that are short numbers, the axis label is now being pushed out of the visible area. Might need to adjust padding or something

Comment (gabemart) [reply client/src/components/charts/LineChart.jsx]: image

**Comment by gabemart** [comment]
@finnfiddle margin for line chart looks good at medium display size

image

At medium display size (right before the break point where we increase the proportional size of all the viz in the dashboard) there's still some overlap

image

This is on the smallest viz size (which is 1/3 the width of the dashboard) at a viewport width of around 1100px

at large display sizes, the offset between the label and the axis is quite large

image

I don't want to get too hung up on absolute perfection because I'm sure we could spend forever tweaking this, but @finnfiddle how much work would it be to decrease the offset a bit at large render sizes and increase it a big at small render sizes?

@nadiagorchakova @janagombitova thoughts?

**Comment by gabemart** [comment]
@finnfiddle I made a change for bar charts

before:
image

after:
image

**Comment by gabemart** [comment]
@finnfiddle I am OK to merge this now. Could you please test out my bar chart margin change, then merge if you approve?

**Pull request merged**